src/targets/gpu/CMakeLists.txt

```diff
@@ -282,3 +282,10 @@ rocm_install_targets(
     INCLUDE
     ${CMAKE_CURRENT_SOURCE_DIR}/include
 )
+
+if(WIN32)
+    add_custom_command(TARGET migraphx_gpu POST_BUILD
+        COMMAND ${CMAKE_COMMAND} -E copy  "${MIOPEN_LOCATION}" $<TARGET_FILE_DIR:migraphx_gpu>
```

Comment:
Where is `MIOPEN_LOCATION` defined? MIOpen's package doesnt define this variable. The only location is the `miopen_DIR` variable that points to the cmake files.